server.js

```diff
@@ -1,38 +1,159 @@
-import express from 'express'
-import cors from 'cors'
-import mongoose from 'mongoose'
+import express from 'express';
+import cors from 'cors';
+import mongoose from 'mongoose';
+import listEndpoints from 'express-list-endpoints';
+import netflixData from './data/netflix-titles.json';
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
 // import goldenGlobesData from './data/golden-globes.json'
 // import avocadoSalesData from './data/avocado-sales.json'
 // import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
+
 // import topMusicData from './data/top-music.json'
 
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo"
-mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
-mongoose.Promise = Promise
+const mongoUrl =
+	process.env.MONGO_URL || 'mongodb://localhost/project-mongo-netflix';
+mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true });
+mongoose.Promise = Promise;
+
+// Defines the port the app will run on
+const port = process.env.PORT || 8080;
+const app = express();
+
+//A show database model to start populating the database
+//shows all the existing data
+const Show = mongoose.model('Show', {
+	show_id: Number,
+	title: String,
+	director: String,
+	cast: String,
+	country: String,
+	date_added: String,
+	release_year: Number,
+	rating: String,
+	duration: String,
+	listed_in: String,
+	description: String,
+	type: String,
+});
+
+//Function to start seeding the database,
+//and it will only run when variable is present and true
+if (process.env.RESET_DB) {
+	console.log('Resetting database!');
+	const seedDataBase = async () => {
+		//Deletes pre-existing shows to prevent duplicates
+		await Show.deleteMany({});
+
+		//Creates a new show
+		netflixData.forEach((item) => {
+			const newShow = new Show(item);
+			newShow.save();
+		});
+	};
 
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
-const port = process.env.PORT || 8080
-const app = express()
+	seedDataBase();
+}
 
 // Add middlewares to enable cors and json body parsing
-app.use(cors())
-app.use(express.json())
+app.use(cors());
+app.use(express.json());
+
+// Our own middlewarethat checks if the database is connected before going to our endpoints
+app.use((req, res, next) => {
+	if (mongoose.connection.readyState === 1) {
+		next();
+	} else {
+		res.status(503).json({ error: 'Service unavailable' });
+	}
+});
+
+//RESTful route
 
 // Start defining your routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
-})
+	res.send('Hello world, welcome to netflix titles API by Madelene Trang');
+});
```

Comment:
Maybe in the welcome message you could add "to see all endpoints, add "/endpoints" at the end of the browser tab" - might make it easier for the user to use the API